lib/resources/etc_hosts_allow_deny.rb

```diff
@@ -65,63 +65,27 @@ def parse_conf(content)
     end
 
     def parse_line(line)
-      client_list = ''
-      options = ''
+      daemons, clients_and_options = line.split(/\s+:\s+/, 2)
+      daemon_list = daemons.split(',').map(&:strip)
 
-      daemon = line.split(':')[0].strip
+      clients_and_options ||= ''
+      clients, options = clients_and_options.split(' : ', 2)
+      client_list = clients.split(',').map(&:strip)
 
-      # If the line contains an ipv6 address, parse using a different
-      # algorithm. ipv6 addresses will containt a '['
-      if !line.index('[').nil?
+      options ||= ''
+      options_list = options.split(': ').map(&:strip)
+
+      puts "Daemons: #{daemon_list.inspect}"
```

Comment:
These should be removed. I provided them to you in order to provide output to show how my method worked but they should not be included in your proposed pull request.